lib/createWorkers.js

```diff
@@ -48,10 +50,37 @@ function createWorkers(handlers, config, options = {}) {
 
   return {
     listen: co.wrap(_listen),
-    close: co.wrap(_close)
+    close: co.wrap(_close),
+    get emitter() {
+      return emitter;
+    },
+    wait(key) {
+      return new Promise(resolve => emitter.once(key, resolve));
+    }
   };
 
   /**
+   * Emit an event with the provided name.
+   *
+   * Emission is scheduled with process.nextTick to allow current work to complete.
+   *
+   * @param {string} name the name of the event to publish
+   * @returns {void}
+   */
+  function emit(name) {
```

Comment:
I would have used the same parameter name as the one for wait(...) : name, eventName, event...